jina/proto/jina.proto

```diff
@@ -106,6 +106,9 @@ message DocumentProto {
     // the weight of this document
     float weight = 5;
 
+    // total number of siblings of this document (docs that are in the same granularity and have the same parent_id)
+    uint32 siblings = 6;
+
```

Comment:
Please use a new place instead of reusing the previous one. This will lead a real breaking change.
```suggestion
    // total number of siblings of this document (docs that are in the same granularity and have the same parent_id)
    uint32 siblings = 24;

```